admin/views/tool-file-editor.php

```diff
@@ -48,7 +48,7 @@
 
 if ( isset( $_POST['submithtaccess'] ) ) {
 	if ( ! current_user_can( 'edit_files' ) ) {
-		die( __( 'You cannot edit the .htaccess file.', 'wordpress-seo' ) );
+		die( esc_html__( 'You cannot edit the .htaccess file.', 'wordpress-seo' ) );
```

Comment:
I know this is not the scope of this pr, but I think we should get `.htaccess` out of this string

Reply:
Done.